lib/api/src/grpc/proto/points.proto

```diff
@@ -48,7 +48,8 @@ message SetPayloadPoints {
   string collection_name = 1; // name of the collection
   optional bool wait = 2; // Wait until the changes have been applied?
   map<string, Value> payload = 3; // New payload values
-  repeated PointId points = 4; // List of point to modify
+  reserved 4; //deprecated List of point to modify
```

Comment:
We can't remove fields in one commit, otherwise client from previous version will be incompatible and we won't be able to support version upgrade without a downtime. I propose to keep deprecated field active with according commentary and only remove it after the next version release.

Reply:
Makes sense :+1:  ill fix it :)